docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -2,23 +2,48 @@
 title: Keptn for Non-Kubernetes Applications
 description: Using Keptn with Non-Kubernetes Applications
 weight: 95
-hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-It is possible to trigger Keptn Tasks for workloads and applications that are deployed outside of Kubernetes.
+Keptn Tasks can be triggered for workloads and applications
+that are deployed outside of Kubernetes.
+For example, Keptn could trigger load and performance tests
+for an application that is deployed on a virtual machine.
 
-For example, to trigger a load test for an application deployed on a virtual machine.
+To do this:
 
-You will still need to deploy Keptn on a Kubernetes cluster, but this can be a very lightweight, single-node kind cluster.
-Keptn's only job is to trigger on-demand tasks, so resource utilization will be minimal.
+1. [Install and enable a Kubernetes cluster](#install-and-enable-a-kubernetes-cluster)
+1. [Create a KeptnTaskDefinition](#create-a-keptntaskdefinition)
+1. [Create and apply a KeptnTask](#create-and-apply-a-keptntask)
+1. [Re-run the KeptnTask](#re-run-the-keptntask)
 
-## Step 1: Create a KeptnTaskDefinition
+## Install and enable a Kubernetes cluster
 
-When you have Keptn installed, [create a KeptnTaskDefinition](../implementing/tasks/).
+You must still
+[install](../install/install.md/#use-helm-chart)
+and
+[enable](../install/install.md/#enable-keptn-for-your-cluster)
+Keptn on a Kubernetes cluster,
+but this can be a very lightweight, single-node KinD cluster; see
+[Create local Kubernetes cluster](../install/k8s.md/#create-local-kubernetes-cluster).
+Keptn only triggers on-demand `KeptnTask` resources
+so resource utilization is minimal.
 
-A `KeptnTaskDefinition` defines **what** you want to execute.
+TODO: How is this cluster associated with the VM
```

Comment:
```suggestion

```
I would remove this for now

Reply:
I can do that -- would love to get this much merged. actually.  But are we concerned that the instructions for doing this may be incomplete?

Reply:
I would avoid trying to explain more because the "remote" setups could differ so much that it does not make sense for us to even try... we give you a place where to run anything you like (aka a container), so you can use it to connect and run stuff on anything you want as long as you know how to do it...